optaplanner-docs/src/main/asciidoc/Chapter-Quick_start/Chapter-Quick_start.adoc

```diff
@@ -355,7 +225,161 @@ public class CloudBalance ... {
+                "org/optaplanner/examples/cloudbalancing/solver/cloudBalancingSolverConfig.xml");
+        Solver<CloudBalance> solver = solverFactory.buildSolver();
+
+        // Load a problem with 400 computers and 1200 processes
+        CloudBalance unsolvedCloudBalance = new CloudBalancingGenerator().createCloudBalance(400, 1200);
+
+        // Solve the problem
+        CloudBalance solvedCloudBalance = solver.solve(unsolvedCloudBalance);
+
+        // Display the result
+        System.out.println("\nSolved cloudBalance with 400 computers and 1200 processes:\n"
+                + toDisplayString(solvedCloudBalance));
+    }
+
+    ...
+}
+----
+====
+
+The code example does the following:
+
+. Build the `Solver` based on a solver configuration (in this case <<solverConfigurationByXML,an XML file>>, `cloudBalancingSolverConfig.xml`, from the classpath).
++ 
+Building the `Solver` is the most complicated part of this procedure. For more detail, see <<cloudBalancingSolverConfiguration, Solver Configuration>>.
+
++
+
+[source,java,options="nowrap"]
+----
+        SolverFactory<CloudBalance> solverFactory = SolverFactory.createFromXmlResource(
+                "org/optaplanner/examples/cloudbalancing/solver/cloudBalancingSolverConfig.xml");
+        Solver solver<CloudBalance> = solverFactory.buildSolver();
+----
+
+. Load the problem. 
++
+`CloudBalancingGenerator` generates a random problem: you will replace this with a class that loads a real problem, for example from a database.
++
+
+[source,java,options="nowrap"]
+----
+        CloudBalance unsolvedCloudBalance = new CloudBalancingGenerator().createCloudBalance(400, 1200);
+----
+
+. Solve the problem.
++
+
+[source,java,options="nowrap"]
+----
+        CloudBalance solvedCloudBalance = solver.solve(unsolvedCloudBalance);
+----
+
+. Display the result.
++
+
+[source,java,options="nowrap"]
+----
+        System.out.println("\nSolved cloudBalance with 400 computers and 1200 processes:\n"
+                + toDisplayString(solvedCloudBalance));
+----
+
+
+[[cloudBalancingSolverConfiguration]]
+=== Solver Configuration
+
+Take a look at the solver configuration:
+
+.cloudBalancingSolverConfig.xml
+====
+[source,xml,options="nowrap"]
+----
+<?xml version="1.0" encoding="UTF-8"?>
+<solver>
+  <!-- Domain model configuration -->
+  <scanAnnotatedClasses/>
+
+  <!-- Score configuration -->
+  <scoreDirectorFactory>
+    <easyScoreCalculatorClass>org.optaplanner.examples.cloudbalancing.optional.score.CloudBalancingEasyScoreCalculator</easyScoreCalculatorClass>
+    <!--<scoreDrl>org/optaplanner/examples/cloudbalancing/solver/cloudBalancingScoreRules.drl</scoreDrl>-->
+  </scoreDirectorFactory>
+
+  <!-- Optimization algorithms configuration -->
+  <termination>
+    <secondsSpentLimit>30</secondsSpentLimit>
+  </termination>
+</solver>
+----
+====
+
+This solver configuration consists of three parts:
+
+. **Domain model configuration**
+
+What can Planner change?::
```

Comment:
There are changes in this big green blob, such as these ::, but I can't see them on github review... How shall we deal with this?